src/lib/api.js

```diff
@@ -37,6 +37,23 @@ export const getItemsFromList = async (token) => {
   const itemsFromList = list.data().items;
 
   if (itemsFromList) {
+    itemsFromList.sort((a, b) => {
+      let fa = a.name.toLowerCase(),
+        fb = b.name.toLowerCase();
+
+      if (fa < fb) {
+        return -1;
+      }
+      if (fa > fb) {
+        return 1;
+      }
+      return 0;
+    });
+
+    itemsFromList.sort((a, b) => {
```

Comment:
que te parece si se deja en una sola linea :D 

` itemsFromList.sort((a, b) => a.howSoon - b.howSoon });`

Reply:
Hola chicos, tengo una cosita por preguntar: 

'- En esta imagen veo que no se dibuja la clase de acuerdo a diferentes tipos de how soon, puse productos con el nombre de soon o not soon para identificar el caso de how soon en estos productos y se dibuja mejor,
![Screenshot_8](https://user-images.githubusercontent.com/63525613/154747422-83dbd6bf-e710-4c3e-a0d4-ac09f4ebd5ba.png)

Reply:
Hola @elizabethriver Eli:

Los rangos que determinan los colores cambiaron a petición de este requerimiento. Te explico:

### Antes
Los posibles estados de un artículo eran los siguientes:
* Soon: 7 días
* Kind of soon: 14 días
* Not soon: 7 días


### Ahora
Los posibles estados de un artículo son los siguientes:
* Necesita comprarlo pronto (menos de 7 días)
* Necesita comprarlo más o menos pronto (entre 7 y 30 días)
* No necesita comprarlo pronto (más de 30 días)
* Inactivo (cuando sólo hay una compra en la base de datos o la compra fue hace mucho tiempo [el tiempo transcurrido desde la última compra es el doble del estimado])


Todo producto que se crea nace con 7, 14 o 30 así que todos (al menos en un primer momento) caen en el rango de 'más o menos '. Luego de las compras y de que cambien sus `howSoon` a otros valores se reflejará más fácilmente el cambio de color.

Reply:
gracias Jairo por la explicacion, he visto con los ultimos cambios esto,
![Screenshot_3](https://user-images.githubusercontent.com/63525613/154996068-b2deed35-5e3f-4153-84ef-5a1c1b813e6d.png)
Orden alfabetico, e inactivo al final